src/controllers/email-controller.js

```diff
@@ -0,0 +1,58 @@
+const Accounts = require('../models/accounts');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const Accounts = require("../models/accounts");
```